upgrade/scripts/upgrade/util/nls-keycloak-configure.py

```diff
@@ -0,0 +1,98 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+import argparse
+import sys
+import requests
+import json
+import urllib3
+
+def globalErrorHandler(error):
+    print(e)
+    sys.exit("something went wrong")
+
+urllib3.disable_warnings()
+
+parser = argparse.ArgumentParser(description='utility script to configure keycloak for cray-nls')
+parser.add_argument('--kc-client-id',required=True, help='keycloak admin client id')
+parser.add_argument('--kc-username',required=True, help='keycloak admin username')
+parser.add_argument('--kc-password',required=True, help='keycloak admin password')
```

Comment:
Would be better not to have this passed on the command line, but instead read from an environment variable.

Reply:
i don't think that's necessary. we ssh as root and only root can get this credentials anyways because you won't have access to kubeconfig file

Reply:
i am assuming we are trying to protect plain text password here? if that is the case, even passing with OS envar, you can still just `strings /proc/PID/environ` to get the password out